app/shared/ui-wizard/src/commonMain/kotlin/ui/wizard/FirstScreenScene.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2024-2025 OpenAni and contributors.
+ *
+ * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ * Use of this source code is governed by the GNU AGPLv3 license, which can be found at the following link.
+ *
+ * https://github.com/open-ani/ani/blob/main/LICENSE
+ */
+
+package me.him188.ani.app.ui.wizard
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.layout.windowInsetsPadding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Button
+import androidx.compose.material3.LocalTextStyle
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import me.him188.ani.app.ui.foundation.layout.AniWindowInsets
+
+/**
+ * 首次启动 APP 的欢迎界面, 在向导之前显示.
+ */
+@Composable
+fun FirstScreenScene(
+    modifier: Modifier = Modifier,
+    contactActions: @Composable () -> Unit,
+    windowInsets: WindowInsets = AniWindowInsets.forPageContent(),
+    onLinkStart: () -> Unit,
```

Comment:
```suggestion
    onClickStart: () -> Unit,
```

参数顺序不对, onClickStart 放在 modifier 之前, contactActions 放在 onClickStart 之后
- onClickStart
- contactActions
- modifier
- - windowInsets